raiden/blockchain/exceptions.py

```diff
@@ -16,15 +16,6 @@ class UnknownRaidenEventType(RaidenUnrecoverableError):
     """
 
 
-class UnknownExternalEventType(RaidenRecoverableError):
```

Comment:
This is still kind of helpful, as design advice/documentation. If it needs to live a dead life in the code is debatable though.